.github/workflows/validate-spectral.yml

```
@@ -39,6 +39,9 @@ jobs:
 
       - name: Install Spectral-CLI
         run: npm install @stoplight/spectral-cli
```

```suggestion
        run: npm install @stoplight/spectral-cli rollup
```

We can do both in the single install - no need for an additional step.